moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryStatusUpdate.js

```diff
@@ -143,6 +145,12 @@ const ProjectSummaryStatusUpdate = ({ projectId, data, refetch, classes }) => {
     setStatusUpdateAddNew(false);
   };
 
+  /**
+   * Only allow the user who wrote the status to edit it
+   */
+  const editableComment =
+    userId === parseInt(getStatusUpdate("added_by_user_id"));
```

Comment:
Sorry Tilly, I take it back. This is what I get for not fully expanding the code I was reviewing. 

I thought the userId declared above was global to the whole component, I didnt realize it was declared within a function. Which is why it's now giving us a `Line 152:5:  'userId' is not defined` error.